frictionless/detector/detector.py

```diff
@@ -171,7 +171,9 @@ def detect_resource(self, resource: Resource) -> None:
             name = os.path.commonprefix(names)
             name = helpers.slugify(name, regex_pattern=r"[^-a-z0-9._/]")
             name = name or "name"
-            scheme, format = helpers.parse_scheme_and_format(resource.path)
+            scheme, format = helpers.parse_scheme_and_format(
```

Comment:
We can't do this, unfortunately, because the correct scheme still will be `file` not `https`.

The fix should around the place where we create a loader like `schemes/local` or `schemes/remote`. Please let me know if you need help I can find it